### DISCUSSION TITLE
SkyBlock Level: Mythic Chocolate Rabbits

### DISCUSSION THREAD

**Discussion by MattTheCuber** [HypixelDev/PublicAPI#661] [Q&A] [answered]
Hello,

Simply stated, there are currently 11 mythic chocolate rabbits in the game. 8 of these give 10 SkyBlock level experience each, 3 do not. We are nearly certain that one of these excluded rabbits is the Zest Zephyr. What are these other 2 excluded mythic chocolate rabbits?

**Comment by lunaynx** [comment]
I'm pretty sure you get full XP for just getting any 8 of the 11 rabbits.

**Reply by lunaynx** [reply, marked as answer]
Update: This is not the case anymore, each one gives XP now.
